modules/cnf-image-based-upgrade-installing-lifecycle-agent-using-web-console.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/cnf-preparing-for-image-based-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-lifecycle-agent-using-web-console_{context}"]
+= Installing the {lcao} by using the web console
+
+You can use the {product-title} web console to install the {lcao}.
+
+.Prerequisites
+
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. In the {product-title} web console, navigate to *Operators* → *OperatorHub*.
+. Search for the *{lcao}* from the list of available Operators, and then click *Install*.
+. On the *Install Operator* page, under *A specific namespace on the cluster* select *openshift-lifecycle-agent*. Then, click Install.
```

Comment:
Just want to check if you need to click install twice or if the Then, click Install at the end of this step needs to be removed, since you have it in the final step?

Reply:
Removed "Then, click Install", I'm fairly certain you only need to click install once I just made a mistake here... Thanks for catching it!